preprocessing.py

```diff
@@ -40,5 +44,7 @@ def one_doc_preprocess(ar, list_of_arguments):
     # is above a good way of reading path?
     list_of_names = glob.glob(path)
     list_of_fields = ["TITLE"]
+    dict_of_terms = {}
     for i in list_of_names:
-        one_doc_preprocess(i, list_of_fields)
+        one_doc_preprocess(i, list_of_fields, dict_of_terms)
+    print(dict_of_terms, file=open("all_terms_in_dict", "w"))
```

Comment:
did you try to deserialize it?
this is just string representation of Python-Map
okay... try to write two methods - serializeIndex(map, filepath) and deserializeIndex(filePath) that returns serialized index
and use these methods
we need more abstraction!!!!)